sdk/data/aztables/transactional_batch.go

```diff
@@ -301,20 +301,24 @@ func (t *Client) generateEntitySubset(transactionAction *TransactionAction, writ
 		return fmt.Errorf("entity properties must contain a %s property", rowKey)
 	}
 	// Consider empty ETags as '*'
-	if transactionAction.IfMatch == nil {
-		star := azcore.ETagAny
-		transactionAction.IfMatch = &star
-	}
+	// if transactionAction.IfMatch == nil {
+	// 	star := azcore.ETagAny
+	// 	transactionAction.IfMatch = &star
+	// }
 
 	switch transactionAction.ActionType {
 	case Delete:
+		ifMatch := "*"
```

Comment:
We have that fancy azcore.ETagAny constant for this, right?

Reply:
Yes. The generated code takes a string, not the azcore.ETag type so this was slightly simpler.